### PR TITLE
feat(web): add "open in browser" tip in wechat browser

### PR DESCRIPTION
**Pull request by hyrious**

### CODE REVIEW

File: web/flat-web/src/utils/i18n.ts [hunk 1]
```diff
@@ -19,6 +19,10 @@ void i18next
         interpolation: {
             escapeValue: false, // react already safes from xss
         },
+        detection: {
+            // don't remember user language
+            caches: [],
+        },
```
Comment (BlackHole1) [inline web/flat-web/src/utils/i18n.ts]: split into another PR

```suggestion
        detection: {
            // no recording of user-selected languages
            caches: [],
        },
```

File: web/flat-web/src/pages/JoinPage/JoinPageMobile.tsx [hunk 2]
```diff
@@ -17,6 +18,10 @@ export default function JoinPageMobile({
 }: JoinPageMobileProps): React.ReactElement {
     const { t } = useTranslation();
 
+    const isWeChat = useMemo(
```
Comment (BlackHole1) [inline web/flat-web/src/pages/JoinPage/JoinPageMobile.tsx]:
```suggestion
    const isWeChatBrowser = useMemo(
```

move to the outside of the components

**Review by Cheerego7 — DISMISSED** [review]

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Review by BlackHole1 — APPROVED** [review]
Great

File: web/flat-web/src/utils/i18n.ts (hunk 1 above)
Comment (hyrious) [reply web/flat-web/src/utils/i18n.ts]: #1269
